src/CommunityToolkit.Maui/MauiCommunityToolkitOptions.cs

```diff
@@ -0,0 +1,28 @@
+using CommunityToolkit.Maui.Behaviors;
+using CommunityToolkit.Maui.Converters;
+
+namespace CommunityToolkit.Maui;
+
+/// <summary>
+/// .NET MAUI Community Toolkit Options.
+/// </summary>
+public class MauiCommunityToolkitOptions
+{
+#pragma warning disable CA1822 // Mark members as static
```

Comment:
I'm starting to think that setting `CA1822` to `error` was too aggressive.

Let's update the `.editorconfig` in this PR , changing it to `suggestion`:
```cs
# CA1822: Member does not access instance data and can be marked as static
dotnet_diagnostic.CA1822.severity = suggestion
```